README.md

```diff
@@ -0,0 +1,185 @@
+[![Build Status](https://travis-ci.org/Advanon/pdf-signatures.svg?branch=master)](https://travis-ci.org/Advanon/pdf-signatures)
+
+# PDF-Signatures
+
+Sign PDFs with PKCS7 signatures like a boss!
+
+## Dependencies
+* UNIX-like OS
+* Java runtime >= 1.8
+* Nodejs >= 8.10
+
+## License
+
+This library is distributed under the AGPL license. However, some dependencies
```

Comment:
Any particular reason for using AGPL here? Better to use Apache 2.0 license here, which is generally more open and more commercially friendly.

Reply:
@whummer please reference my comment to Andrea :)
Or I'l simply copy-paste it here:
> tbh I'm not that familiar with AGPL, but if I'm not mistaken, any code which uses AGPL code must be also licensed under AGPL. In our case itext is AGPL-licensed, which forces us to either make our code AGPL as well, or purchase their commercial license.